packages/content-conformance/src/cli.ts

```diff
@@ -19,32 +19,27 @@ yargs(hideBin(process.argv)).command(
       description: 'Path to config file',
       type: 'string',
     },
-    files: {
-      description: 'Path to content files',
-      default: [],
-      type: 'array',
-    },
   },
   async (argv): Promise<void> => {
+    const [...files] = <string[]>argv._
```

Comment:
This allows us to pass rest args. In this case an array of files.

before:
`hc-content content-check --cwd --config --files`

after:
`hc-content --cwd --config [path/to/file]`